packages/frontend/src/hooks/useChartColorConfig.ts

```diff
@@ -5,6 +5,13 @@ import { EChartSeries } from './echarts/useEchartsCartesianConfig';
 
 export type SeriesLike = EChartSeries | Series;
 
+export const isGroupedSeries = (series: SeriesLike) => {
```

Comment:
Has a bit of duplication with the logic below, but opted to keep it like this for clarity.